From 1a8b161da80b93d122854c854c95bf9f7d326cdc Mon Sep 17 00:00:00 2001
From: Allan Sandfeld Jensen <allan.jensen@theqtcompany.com>
Date: Fri, 3 Jul 2015 14:19:02 +0200
Subject: [PATCH] Fix compile error on big endian

DestFormat is not defined here but should be Format_RGBX8888

Change-Id: Ie33989a6fc31650434281a6b7b0f4806524268af
---
 src/gui/image/qimage_conversions.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gui/image/qimage_conversions.cpp b/src/gui/image/qimage_conversions.cpp
index e0ca2c5..02f32aa 100644
--- a/src/gui/image/qimage_conversions.cpp
+++ b/src/gui/image/qimage_conversions.cpp
@@ -1103,7 +1103,7 @@ static bool mask_alpha_converter_rgbx_inplace(QImageData *data, Qt::ImageConvers
         }
         rgb_data += pad;
     }
-    data->format = DestFormat;
+    data->format = QImage::Format_RGBX8888;
     return true;
 #endif
 }
-- 
2.1.4

